sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -468,6 +468,7 @@ object FunctionRegistry {
     expression[TransformKeys]("transform_keys"),
     expression[MapZipWith]("map_zip_with"),
     expression[ZipWith]("zip_with"),
+    expression[AddFields]("add_fields"),
```

Comment:
If you add this in bultin-functions, you need to add tests in `SQLQueryTestSuite`.

Reply:
didn't know that, thanks, I've added the tests now